dom.go

```diff
@@ -110,12 +110,29 @@ type HTML struct {
 	properties, attributes          map[string]interface{}
 	eventListeners                  []*EventListener
 	children                        []ComponentOrHTML
+	new                             bool
 }
 
 // Node returns the underlying JavaScript Element or TextNode.
 func (h *HTML) Node() *js.Object { return h.node.(wrappedObject).j }
 
-func (h *HTML) restoreText(prev *HTML) {
+func (h *HTML) newNode() {
```

Comment:
nit: this doesn't return a new node as e.g. `new` (builtin) would, but rather it is initializing `h.node` so I might suggest `initNode` as being the name. Do you think that'd be a better name too, or disagree?